### DISCUSSION TITLE
@mxmou Dark mode details

### DISCUSSION THREAD

**Discussion by WorldLanguages** [ScratchAddons/ScratchAddons#792] [Discussion]
Can there be a grayish border between comments? (I understand this looks OK if scratch2→3 is turned on, but some people might not use it)
![image](https://user-images.githubusercontent.com/17484114/99836040-41676e00-2b44-11eb-9fb1-4a6bcbac9dbd.png)

Should the text be white?
![image](https://user-images.githubusercontent.com/17484114/99836133-6360f080-2b44-11eb-81c3-f1e0742fe177.png)

Aaaaaaa my eyes (URL: https://scratch.mit.edu/users/gdpr0000001/ comments never load)
![image](https://user-images.githubusercontent.com/17484114/99836153-6d82ef00-2b44-11eb-9517-a61d24d13cd5.png)

cc @mxmou

**Comment by WorldLanguages** [comment]
More little details:

When sending a comment
![image](https://user-images.githubusercontent.com/17484114/99836442-dcf8de80-2b44-11eb-9c4f-8d671001b1ef.png)

Report/delete comment modal:
![image](https://user-images.githubusercontent.com/17484114/99836464-e6824680-2b44-11eb-8234-c1f0ec0e21ee.png)

**Reply by mxmou** [reply]
> Report/delete comment modal:

Another thing that works correctly with 2.0 3.0 but not without it.

**Reply by mxmou** [reply]
> When sending a comment

This one would be hard to fix but I can try.

**Comment by mxmou** [comment]
> Can there be a grayish border between comments? (I understand this looks OK if scratch2→3 is turned on, but some people might not use it)

That would need changes to 2.0 → 3.0 (in particular, `experimental_dark_compatibility.css`). I do want to improve the look of this dark mode when 2.0 → 3.0 is disabled (tabs in My Stuff are currently really strange), but fixing true bugs is more important for now.

> Should the text be white?
> ![image](https://user-images.githubusercontent.com/17484114/99836133-6360f080-2b44-11eb-81c3-f1e0742fe177.png)

Yes, it should be.

> Aaaaaaa my eyes (URL: https://scratch.mit.edu/users/gdpr0000001/ comments never load)

2.0 → 3.0 removes the background, so I didn't notice this bug. Yes, it needs to be fixed.
